src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Condition/IsInitializedPlugin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Communication\Plugin\Oms\Condition;
+
+use Orm\Zed\Sales\Persistence\SpySalesOrderItem;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\Oms\Dependency\Plugin\Condition\ConditionInterface;
+
+/**
+ * @method \SprykerEco\Zed\Computop\ComputopConfig getConfig()
+ * @method \SprykerEco\Zed\Computop\Business\ComputopFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Computop\Communication\ComputopCommunicationFactory getFactory()
+ */
+class IsInitializedPlugin extends AbstractPlugin implements ConditionInterface
+{
+    /**
+     * @api
+     *
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrderItem $orderItem
+     *
+     * @return bool
+     */
+    public function check(SpySalesOrderItem $orderItem)
+    {
+        /** @var \Orm\Zed\Computop\Persistence\SpyPaymentComputopOrderItem $computopOrderItem */
+        $computopOrderItem = $orderItem->getSpyPaymentComputopOrderItems()->getLast();
```

Comment:
For better reading, please create separate functions for such cases.
We do not recommend inline doc-block